extension/src/popup/views/AccountHistory/index.tsx

```diff
@@ -99,13 +104,25 @@ export const AccountHistory = () => {
           operation.type ===
             Horizon.HorizonApi.OperationResponseType.createAccount &&
           operation.account !== publicKey;
+        const isDustPayment =
+          isPayment &&
+          "asset_type" in operation &&
+          operation.asset_type === "native" &&
+          "to" in operation &&
+          operation.to === publicKey &&
```

Comment:
checking for a native payment to the user's account

Reply:
should this be flipped? `operation.asset_type === "native"`

I think dust payments would typically not be xlm right, as the spam is to try to get bunk tokens to show up in the interface by sending dust amounts of them.

Reply:
Actually, according to this [doc](https://docs.google.com/document/d/1tpGsVx3ZlRZbdhedkOfj_cUE2Ixdzct1v9xZP9zj3FE/edit), we're specifically on the lookout for small XLM payments

Reply:
oh cause of the memo, yup got it!